### PR TITLE
Provisioner upgrade test script fix.

### PR DESCRIPTION
**Pull request by ashukakkar**
Signed-off-by: ashukakkar <karan.kakkar@seagate.com>

# Problem Statement
- Upgrade test script fix.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [x] New/Affected tests are executed on Latest Build
-  [x] Attach test execution logs
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: comptests/prov/test_prov_k8s_cortx_upgrade.py [hunk 1]
```diff
@@ -298,6 +296,11 @@ def test_37514(self):
             resp = str(resp, 'UTF-8')
         resp = "".join(resp).replace("\\n", "\n")
         assert_utils.assert_in("Invalid argument provided", resp)
+        LOGGER.info("Remove upgrade pid file for next TC.")
```
Comment (priyankaborawake) [inline comptests/prov/test_prov_k8s_cortx_upgrade.py:299]: This can be either added to setup method (as a precheck) or teardown method.

Comment (ashukakkar) [reply comptests/prov/test_prov_k8s_cortx_upgrade.py:299]: Thanks. Let me add it.

**Comment by ashukakkar** [comment]
Test run html report and console logs are attached in below zip file:
[Upgrade_Test_Run_Report.zip](https://github.com/Seagate/cortx-test/files/8573095/Upgrade_Test_Run_Report.zip)

File: comptests/prov/test_prov_k8s_cortx_upgrade.py (hunk 1 above)
Comment (ashukakkar) [reply comptests/prov/test_prov_k8s_cortx_upgrade.py]: Done. Have put that login in teardown method.

**Review by pragamj — APPROVED** [review]

**Review by ravindrachoudhari — APPROVED** [review]

**Review by priyankaborawake — APPROVED** [review]

**Review by seagate-sarang-sawant — APPROVED** [review]

**Pull request merged**